### ISSUE TITLE
Consider breaking to separate repos

### ISSUE THREAD

**Issue by ChrisChinchilla** [testthedocs/vale-styles#28]
While this isn't ideal for people finding the different styles (though this repo could be a guide to other styles instead), splitting into separate repos allows for the styles to support Vale server better, and for users to add individual styles as submodules to their projects.

**Comment by dylan-k** [comment]
@ChrisChinchilla I'm willing to assist with this if there's any good way to help?

**Comment by ChrisChinchilla** [comment]
Thanks! @dylan-k ! It's relatively easy, I'll create the repos, and then assign some tasks to you? Then there'll be some post-tasks…

**Comment by dylan-k** [comment]
That sounds great.

**Comment by ChrisChinchilla** [comment]
Note, for history preserving, I found this worked - https://medium.com/@ayushya/move-directory-from-one-repository-to-another-preserving-git-history-d210fa049d4b

**Comment by dylan-k** [comment]
@ChrisChinchilla there may be an additional way for, as you said "people finding the different styles." Vale's creators maintain [a collection of styles](https://github.com/errata-ai/styles) and have mentioned [in the docs](https://errata-ai.github.io/vale-server/docs/style): "Have a style you'd like to see implemented? [Let us know! ](https://github.com/errata-ai/styles/issues/new)We're actively looking to expand our library of pre-made, open-source styles."

The requirements for inclusion seem pretty straightforward. "All styles in this library must (1) be maintained in their own (dedicated) repository, (2) publish releases following Semantic Versioning, and (3) include a meta.json ..."

**Comment by ChrisChinchilla** [comment]
Yup, exactly @dylan-k that's what we're working towards, and why we're separating them out first, then the next steps.

> All styles in this library must (1) be maintained in their own (dedicated) repository

Being the first main requirement. I'll recap what we've both done and what still needs to be done asap, first week back at work…

**Comment by dylan-k** [comment]
@ChrisChinchilla The styles you made repos for, a while ago, have all been moved over. Did you want to move any of the others as well?

**Comment by ChrisChinchilla** [comment]
@dylan-k Yes I will, I get sporadic gaps of free time to work on things, I did all these during FOSSDEM ;)

I will also go through everything to create some tasks for the next steps, just testing the process with one or two repos first.
